recipes/m2crypto/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: m2crypto
+  version: !!str 0.23.0
+
+source:
+  fn: M2Crypto-0.23.0.tar.gz
+  url: https://pypi.python.org/packages/source/M/M2Crypto/M2Crypto-0.23.0.tar.gz
+  md5: 89557730e245294a6cab06de8ad4fb42
+
+  patches:
+    - setup.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]
+  skip: True  # [py33 or py34 or py35]
```

Comment:
Could we please combine these and do the following?

```
skip: true  # [not linux or py3k]
```

Based on your comments it sounds like Python 3.x is completely out.